client-common/src/tendermint/types.rs

```diff
@@ -24,7 +24,7 @@ pub use tendermint::{
 };
 
 /// crypto-com instantiated genesis type
-pub type Genesis = GenericGenesis<InitConfig>;
+pub type Genesis = GenericGenesis<Option<InitConfig>>;
```

Comment:
Is this `Option` needed?

Reply:
Before in tendermint-rs, it had a field `Option<AppState>` -- now the type changed to `AppState`, so it'd require Default implementation `AppState` / `InitConfig`.
I think Option may be clearer, as it'll be immediate that the app_state was missing or incorrect in json rather than some arbitrary default value?